Sources/ShortcutRecorder/SRShortcut.m

```diff
@@ -421,7 +421,7 @@ - (void)encodeWithCoder:(NSCoder *)aCoder
 + (instancetype)new
 {
     [self doesNotRecognizeSelector:_cmd];
-    return nil;
```

Comment:
This change is undesirable since we don't want to instantiate any object here. An exception that leads to crash is desirable here since this is a programmer's error.